api/overdrive.py

```diff
@@ -1159,17 +1159,17 @@ def explain_main_collection(self, collection):
         print "%d associated Overdrive Advantage account(s)." % len(
             advantage_accounts
         )
-        for i in advantage_accounts:
+        for collection in advantage_accounts:
```

Comment:
This `collection` is scoped to the for loop, right? Because there is another `collection` in this function’s param.

Reply:
Yeah, this is masking the other `collection` variable. I'll change the name to make it more clear.